R/apply_theme.R

```diff
@@ -9,6 +9,7 @@
 #' @param fontfamily string with the name of a supported font
```

Comment:
fontsizes: list for sizes for different options is a bit obscure. I would not know how many font sizes I need to provide based on this, nor to what it will be applied. Can you be a bit more specific?

Reply:
The same is true for strata. You will need to provide a better explanation

Reply:
Will do :+1: